src/models/Order.js

```diff
@@ -0,0 +1,74 @@
+const mongoose = require('mongoose');
+
+const OrderSchema = new mongoose.Schema(
+  {
+    orderNumber: {
+      type: String,
+      required: true,
+      unique: true,
+    },
+    buyer: {
+      type: mongoose.Schema.Types.ObjectId,
+      ref: 'User', 
+      required: true,
+    },
+    seller: {
+      type: mongoose.Schema.Types.ObjectId,
+      ref: 'User', 
+      required: true,
+    },
+    items: [
+      {
```

Comment:
I think it's better to add the main information about the book (title, author, condition) rather than reference the book. This is because the user might delete the book after selling it.